.github/workflows/rit.yml

```diff
@@ -114,11 +122,12 @@ jobs:
           echo "SAFE_BRANCH_NAME=$SAFE_BRANCH_NAME" >> $GITHUB_ENV
 
       - name: Run Rootstock Integration Tests
-        uses: rsksmart/rootstock-integration-tests@497172fd38dcfaf48c77f9bb1eeb6617eef5eed6 #v1
+        uses: rsksmart/rootstock-integration-tests@feature/adding_repo_owner_input
```

Comment:
suggestion:

Let's wait the RIT PR be merged, so a new release is generated and we use it here.

Reply:
sure, I'm going to drop that commit once the RIT pr is merged. I added that for testing purposes